### ISSUE TITLE
What would be the workflow for bulk uploads?

### ISSUE THREAD

**Issue by riverfr0zen** [plagmada/plagmada-archives#3]
Please note that this issue concerns the workflow itself (from upload to publish) for image content when bulk uploaded, and NOT the bulk upload tool. A separate issue should be created for that.

Initial suggestion:
1. Bulk-upload images through tool / interface
2. Creates corresponding Omeka items with minimal data populated (probably just URL), in an 'unpublished' state.
3. Site curator(s) then fill in missing metadata (titles, descriptions, attributions, etc. etc.) on a per-image basis.
4. Site curator(s) publish completed item(s)

**Comment by kwhite2** [comment]
Hrm, @riverfr0zen, can you tell me a bit more about the workflow you've proposed here? I'm assuming that it's because you're going to program a bulk tool (as discussed elsewhere) to get the content from the current site into Omeka?

**Comment by riverfr0zen** [comment]
Yes, it is related to the other issue on developing a bulk tool. That issue was created separately so that we could discuss software dev research on creating a new tool. However, this issue is where we should discuss the ideal workflow (and hopefully come to a final decision). I'm attaching your comment from the previous thread so that we can discuss the workflow involved with the Omeka plugin you mentioned:

> kwhite2 commented 13 minutes ago
> I may be behind the curve here, but there is a batch upload function in one of the Omeka plugins that's REALLY simple to use. If we can get the metadata and urls of the file locations output into a CSV file, it can all be really easily mapped. Is the Omeka admin login communal? If not, can someone set me up in there as an admin? Thanks!

**Comment by riverfr0zen** [comment]
@kwhite2 I will get you a login for the Omeka instance sometime today or tomorrow so you can review what is there now.

**Comment by riverfr0zen** [comment]
@kwhite2 In case you did not get a notification from Slack, I've messaged you with admin access info to the test Omeka instance. Let me know if you can't find the message.
